### ISSUE TITLE
Web GUI cannot retrieve redis keys with backslash character (\) in it

### ISSUE THREAD

**Issue by williamBoily** [joeferner/redis-commander#421]
Hi, I believe there is a bug in the web GUI when the redis keys contains backslash `\`.

The key tree stop "expanding" when it encounters `\` in the key name.

I'm using redis-commander in a container with the `latest` tag

Here is an example where the keys stop showing after the `\\`
original keys : 
```
PHPREDIS_SESSION:storage::plugin::690b8981\\1::A
PHPREDIS_SESSION:storage::plugin::690b8981\\1::B
PHPREDIS_SESSION:storage::plugin::690b8981\\1::C
```
See in the screen shot the different ways tried to retrieve the keys but only when removing the `\` it worked.
![Screen Shot 2020-11-10 at 7 47 52 PM](https://user-images.githubusercontent.com/32312693/98751277-cf499900-238d-11eb-8a75-e90d4d732ec0.png)

To compare with the redis-cli, I can retrieve the keys by escaping the backslash.
![image](https://user-images.githubusercontent.com/32312693/98751760-b8577680-238e-11eb-848f-766f4992bccf.png)

**Comment by sseide** [comment]
yes, you are right, seems nobody used backslashes before... i'll have a look at it. 
May be a problem with double qoutes needed.

**Comment by sseide** [comment]
new docker image will be available soon. Can you retest and verify its working now?

**Comment by williamBoily** [comment]
I confirm it is working now!
Thank you for the incredibly fast fix!

Well done 🎉

**Comment by sseide** [comment]
ok - thanks. I leave it open until a new fixed version is released to NPM.

**Comment by LastDragon-ru** [comment]
> I leave it open until a new fixed version is released to NPM

@sseide seems not yet released?
